app/scripts/models/reliers/fx-desktop.js

```diff
@@ -78,6 +80,10 @@ define([
      */
     isCustomizeSyncChecked: function () {
       return !! (this.isSync() && this.get('customizeSync'));
+    },
+
+    isSignupDisabled: function () {
+      return this.get('exclude_signup') === '1';
     }
```

Comment:
Aye, I'd like for this `exclude_signup` to disappear in favour of the context as soon as possible, even if the rest of the stuff stick around for a while.